packages/webpack/src/loader.ts

```diff
@@ -38,7 +38,9 @@ function createIdentifier(
     'base64',
   );
 
-  const unloader = `${virtualResourceLoader}?source=${base64}`;
+  const unloader = `${virtualResourceLoader}?source=${encodeURIComponent(
```

Comment:
`this.getOptions` uses node's native `querystring.parse`, which decodes percent-encoded characters for you. Raw base64 ends up malformed after this process, so we pre-encode the base64 string so it ends up decoded correctly on the other end (in the loader).